src/app/widgets/Console/Terminal.jsx

```diff
@@ -220,7 +223,13 @@ class TerminalWrapper extends PureComponent {
 
     writeln(data) {
         this.term.write('\r');
-        this.term.write(data);
+        if (data.toLowerCase().includes('error')) {
```

Comment:
Do you need the double toLowerCase calls here?  ALARM: and error: are constant responses from the firmware and will never have alternative capitalization.  

You can also include the colon after each.